src/masternode-budget.h

```diff
@@ -515,6 +515,19 @@ class CBudgetProposal
 
     void CleanAndRemove(bool fSignatureCheck);
 
+    // If URL doesn't have a scheme, add one
+    std::string GetFixedupURL(std::string scheme="http")
```

Comment:
nit,might be better having scheme as https instead of http

Reply:
So here's my logic for choosing http rather than https:

1. Not all servers have appropriate https (although it's may kick you to http; I didn't test).
2. If they wanted to be specific, they could have specified it with the proposal.
3. It does pop up a window showing them the URL and asking them to confirm it before continuing; so they can copy it and replace it with https.

But this portion (related to Qt) may be changed with the new wallet anyway.

So: noted; and I'll see what comes with the new wallet and fuzzballs direction (as we are in a holding pattern with this for now)